client/app/lib/components/MaterialSummernote.jsx

```diff
@@ -131,6 +134,7 @@ class MaterialSummernote extends React.Component {
         />
         <div className="material-summernote">
           <ReactSummernote
+            ref={(ref) => { this.reactSummernote = ref; }}
```

Comment:
Just wondering, must this be a function, or can this just be `ref={ this.reactSummernote }`?